catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/authentication/UserContext.java

```diff
@@ -125,7 +125,7 @@ public static UserContext getUserContext(
     } else if (authenticationType == AuthenticationType.KERBEROS) {
       // if the kerberos authentication is inherited from the parent context, we will use the
       // parent context's kerberos configuration.
-      if (authenticationConfig.isSimpleAuth()) {
+      if (parentContext != null && authenticationConfig.isSimpleAuth()) {
```

Comment:
In fact, if the value of `authenticationType` is `KERBEROS` and `authenticationConfig.isSimpleAuth()` is true, the value of `parentContext` is not null.  Anyway, adding a check won't make things worse.